collator.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM phusion/baseimage:jammy-1.0.0
```

Comment:
Would it not be easier to use alpine or something more lightweight? Do we need ubuntu?

This would mean we can use official images too.

Reply:
alpine and musl would make things more complicated